skbeam/core/utils.py

```diff
@@ -1215,6 +1217,51 @@ def geometric_series(common_ratio, number_of_images, first_term=1):
 
     geometric_series = [first_term]
 
-    while geometric_series[-1]*common_ratio < number_of_images:
-        geometric_series.append(geometric_series[-1]*common_ratio)
+    while geometric_series[-1] * common_ratio < number_of_images:
+        geometric_series.append(geometric_series[-1] * common_ratio)
     return geometric_series
+
+
+def bin_grid(image, r_array, pixel_sizes, statistic='mean', mask=None,
```

Comment:
yes, I was thinking actually maybe superseding this by adding a `rpix` parameter to the `RadialBinnedStatistic`. Then maybe refactor your function to just do the autobinning, which isn't done anywhere else and essential. Also a little busy, but should submit something for discussion later.